Makefile

```diff
@@ -0,0 +1,9 @@
+
+.PHONY: test
+test: tools
```

Comment:
I'd have a separate Make goal for the unit tests, and another one for the e2e, so that it's possible to call them in separated stages of the build pipeline

Reply:
Done